### DISCUSSION TITLE
Add new get reviews endpoint for shop owners that includes transaction_id in response

### DISCUSSION THREAD

**Discussion by switmac** [etsy/open-api#669] [Ideas]
I wanted to match the user review on this specific sale is there a way for me to achieve this? 

Can we request to have get Listing Review API return the transaction_id?

**Comment by Gareth064** [comment]
Hi @switmac. The Reviews endpoints are "open" to anyone with an API key to use.
A transaction ID is shop specific data which would mean the endpoints would have to be locked down to only OAuth Token calls, which would break anybody elses app who provides a service where the show reviews on any listing (not just ones belonging to a specific shop)

Maybe the suggestion would be to provide another endpoint for shop owners to download their own reviews, which could then include data, like transaction_id, more specific to their shop.

**Reply by switmac** [reply]
yes that would be a good alternative @Gareth064 for Etsy to add new endpoint for the shop owned reviews. but will Etsy consider adding such endpoint @etsyachristensen please?

**Reply by etsyachristensen** [reply]
@switmac Can you repost that as a new discussion in the Ideas category? That way it's clearer what the request is and allows folks to vote on it there.

**Reply by etsyachristensen** [reply]
Actually, disregard that. I'll move it to Ideas and rename if I can.
